src/main/scala/scalismo/faces/sampling/face/proposals/SphericalHarmonicsLightProposals.scala

```diff
@@ -251,7 +250,7 @@ object SphericalHarmonicsLightProposals {
           if (math.abs(diff.x - diff.y) + math.abs(diff.y - diff.z) < monochromaticityThreshold) { //if monochromatic change, then we could take any channel, we choose the first one.
             totProb += 3.0 * (-0.5 * math.pow(diff.x / sdev, 2) - 0.5 * math.log(2.0 * math.Pi) - math.log(sdev))
           } else {
-            return Double.NegativeInfinity //if non monochromatic change
+            returning(Double.NegativeInfinity) //if non monochromatic change
```

Comment:
what does returning mean? Shouldn't this just be `Double.NegativeInfinity`?

Reply:
There was a warning that returning from an nested anonymous function is deprecated since Scala 3. And as far as I understood, the returning is a possible library replacement from scala.util.control.NonLocalReturns.